docs/addons/secrets-store.md

```diff
@@ -0,0 +1,50 @@
+# Secrets Store Add-on
+
+The Secrets Store Add-on provisions the [AWS Secrets Manager and Config Provider for Secret Store CSI Driver(ASCP)](https://docs.aws.amazon.com/secretsmanager/latest/userguide/integrating_csi_driver.html) on your EKS cluster. With ASCP, you now have a plugin for the industry-standard Kubernetes Secrets Store [Container Storage Interface (CSI) Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) used for providing secrets to applications operating on Amazon Elastic Kubernetes Service.
```

Comment:
Tough for me to parse the acronym here. Can we put (ASCP) directly after `AWS Secrets Manager and Config Provider`?

Reply:
addressed